app/instrumentation-tests/src/org/commcare/androidTests/MenuTests.kt

```diff
@@ -24,7 +24,6 @@ import org.junit.runner.RunWith
 
 @RunWith(AndroidJUnit4::class)
 @LargeTest
-@BrowserstackTests
```

Comment:
annotation on class doesn't work ?

Reply:
It works. `BrowserstackTests` means that the tests will run on browserstack. and we don't want to run all the tests from this class on browserstack. So I've annotated the tests which need to run there.

Reply:
Ahh I completely misunderstood the intent here. Thanks for clarification.